packages/generator-caliper/generators/benchmark/index.js

```diff
@@ -104,6 +104,7 @@ module.exports = class extends Generator {
                     {name: 'Fixed Backlog', value: 'fixed-backlog'},
                     {name: 'Linear Rate', value: 'linear-rate'},
                     {name: 'Fixed Feedback Rate', value: 'fixed-feedback-rate'}
+                    {name: 'Fixed Load', value: "fixed-load"}
```

Comment:
Please run the tests locally before submitting a PR. Doing so will quickly catch bugs in your submission. Here the build fails because you put a bug into the code, you are missing a `,` at the end of line 106